pandas/core/generic.py

```diff
@@ -2207,6 +2207,10 @@ def to_json(
 
                 Describing the data, where data component is like ``orient='records'``.
 
+                Note that ``orient='table'`` contains a 'pandas_version' field under
```

Comment:
can you move this to the Notes section (or create if its not there)

Reply:
Done!